source/dpp/runtime/options.d

```diff
@@ -123,6 +149,9 @@ struct Options {
                 "parse-as-cpp", "Parse header as C++", &parseAsCpp,
                 "define", "C Preprocessor macro", &defines,
                 "hard-fail", "Translate nothing if any part fails", &hardFail,
+                "mscrtlib",
```

Comment:
I'd either find a way to version this out only for Windows, or static assert that it's never used on !Windows.

Reply:
Now that the code ignores the flag on non-windows, I think it is fine to leave it there. This allows using a unified set of flags in the build system (Dpp ignores them smartly based on the OS0.

Reply:
I don't think it makes sense to pass in `--mscrtlib` except for Windows. It's at the very least surprising to someone who doesn't know any better, which will be nearly everyone.

Reply:
I'm fine either way. If you know any way to do this, let me know.

Reply:
```d
version(Windows) {}
else
    assert(mscrtlib == "");
```

Reply:
Fixed in 
https://github.com/atilaneves/dpp/pull/285/commits/6ed5475095e7d93e53442aeabdcf8bfa36a367ef